legal-api/src/legal_api/resources/v2/review.py

```diff
@@ -0,0 +1,47 @@
+# Copyright © 2024 Province of British Columbia
```

Comment:
review.py endpoint file is currently in v2 folder, unsure if this should be placed elsewhere.

Reply:
move this to admin since involuntary dissolution is implemented there

Reply:
can you rename the file name to reviews.py